### PR TITLE
Install git through Visual Studio Code

### PR DESCRIPTION
**Pull request by srabhayraj**
## Overview
**TL;DR**
<One to two sentence description of the issue you are encountering or trying to solve.>

<Link to related issue. Type `closes #RELATEDISSUENUMBER` to establish a link.>

### Questions
<If relevant, write a list of questions that you would like to discuss related to the changes that you have made.>

### Next Steps
<If incomplete, create a task list of items that are still being worked on within the Pull Request.>

### Review
<If complete, or ready for :eyes:, use @mentions for quick questions, specific feedback, and progress updates.>

### CODE REVIEW

File: git-guides/install-git.md [hunk 1]
```diff
@@ -26,6 +26,20 @@ Installing GitHub Desktop will also install the latest version of Git if you don
 
 Note: [`git-scm`](https://git-scm.com/download/win) is a popular and recommended resource for downloading Git for Windows. The advantage of downloading Git from `git-scm` is that your download automatically starts with the latest version of Git included with the recommended command prompt, `Git Bash` . The download source is the same [Git for Windows installer](https://gitforwindows.org/) as referenced in the steps above.
 
+## Install Git on Windows thrpugh Visual Studio Code
```
Comment (brianamarie) [inline git-guides/install-git.md:29]:
```suggestion
## Install Git on Windows through Visual Studio Code
```

**Review by brianamarie — CHANGES_REQUESTED** [review]
Hi @srabhayraj! Thank you so much for this contribution. I've made one suggestion for a spelling error. Once this is corrected, I think this is good to merge!

**Comment by srabhayraj** [comment]
Spelling Change Updation

**Comment by srabhayraj** [comment]
> ## Overview
> **TL;DR**
> ### Questions
> <If relevant, write a list of questions that you would like to discuss related to the changes that you have made.>
> 
> ### Next Steps
> <If incomplete, create a task list of items that are still being worked on within the Pull Request.>
> 
> ### Review
> <If complete, or ready for 👀, use @mentions for quick questions, specific feedback, and progress updates.>
[Pull Request Answers.txt](https://github.com/github/training-kit/files/5157745/Pull.Request.Answers.txt)
